client/src/components/new-request/submit-request/reserve-submit.vue

```diff
@@ -118,7 +124,18 @@ export default class ReserveSubmitButton extends Vue {
 }
 </script>
 
-<style scoped lang="sass">
-.margin-top-auto
-  margin-top: auto !important
+<style scoped lang="scss">
+.margin-top-auto {
+  margin-top: auto !important;
```

Comment:
Use Vuetify class instead.
https://v2.vuetifyjs.com/en/styles/spacing/#how-it-works

Reply:
For sure, will do!

Reply:
It already had the Vuetify Syntax in there alongside this custom class! 
Removed this 👍